B/Blosc2/build_tarballs.jl

```diff
@@ -0,0 +1,51 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Blosc2"
+version = v"2.0.0-rc1"
```

Comment:
We prefer not to use anything outside of `X.Y.Z`, it makes the package manager a bit unhappy.  I think 2.0.0 is fine, when there will be the final release we'll do a new build which will bump the build number

Reply:
ok done